### ISSUE TITLE
Page reload casing logout

### ISSUE THREAD

**Issue by damascene** [nosdav/pastebin#21]
When I refresh (reload) the pastebin webpage I'm logged out. I need to login again to be able to save or load files.

**Comment by melvincarvalho** [comment]
Thanks for raising.  Yes, I think we should improve this UX.

So by default it should remember a logged in user?

**Comment by damascene** [comment]
Yeah I guess so. I don't remember many applications that are requiring a login after refresh.
